src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -403,26 +407,26 @@ const GovernanceReviewTeam = () => {
             <Route
               path="/governance-review-team/:systemId/actions/provide-feedback-need-biz-case"
               render={() => (
-                <SubmitAction
-                  action="PROVIDE_FEEDBACK_NEED_BIZ_CASE"
+                <ProvideGRTFeedback
+                  query={AddGRTFeedbackRequestBizCaseQuery}
```

Comment:
Niiiiice! I like how we're able to still get some code reuse here while keeping the queries discrete.